servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -1108,6 +1108,195 @@ void TextureStorage::texture_proxy_initialize(RID p_texture, RID p_base) {
 	tex->proxies.push_back(p_texture);
 }
 
+// Note: We make some big assumptions about format and usage. If developers need more control,
+// they should use RD::texture_create_from_extension() instead.
```

Comment:
Is it correct this comment refers to `texture_create_from_extension` ?

Reply:
Yep! This PR adds `RenderingServer::texture_create_from_native_handle()` which is implemented by both OpenGL and the rendering device renderers, but if you're using rendering device you can get more control by using the pre-existing `RenderingDevice::texture_create_from_extension()` which has more options. I put a similar note in the docs.